config/routes.rb

```diff
@@ -119,6 +119,10 @@
       constraints: { section: /(jobseeker-guides|get-help-hiring)/, post_name: /[\w-]+/ },
       as: :post
 
+  get "/:section" => "posts#index",
+      constraints: { section: /(jobseeker-guides|get-help-hiring)/ },
```

Comment:
This will also match e.g. `/spooky-jobseeker-guides-for-ghosts`.

Reply:
Ah right, well spotted - do we need `^jobseeker-guides$|^get-help-hiring$` ?

Reply:
@robyoung91 Turns out I was wrong, they already anchor them (and in fact the anchors will break it):
https://guides.rubyonrails.org/routing.html#segment-constraints

You can change it to `get "/:section" => "posts#index", section: /(jobseeker-guides|get-help-hiring)/` to be more compact if you want but no need if you don't want to.